Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.ts

```diff
@@ -40,7 +42,9 @@ export class NavigationComponent implements OnInit, OnDestroy {
     private router: Router,
     public dialog: MatDialog,
     public i18n: I18n,
-    public featureFlagService: FeatureFlagService
+    public featureFlagService: FeatureFlagService,
+    public badge: MatBadgeModule,
```

Comment:
is badge really used?

Reply:
It's used at the *person-icon* and at *Release Notes*

Reply:
but it is not referenced, you only have to import Modules in the import statement of other modules. In *.ts files you only have to import things you are referencing in code, e.g. services

Reply:
?